package.json

```diff
@@ -24,7 +24,8 @@
   },
   "icon": "icon.png",
   "engines": {
-    "node": "*"
+    "node": ">=16.0",
```

Comment:
v14 is still active and even v12 is an LTS. So I wouldn't set this too high. Also, this is clearly a separate change so I expect it to be done in a separate PR.